delphi/log/result_analysis.py

```diff
@@ -3,11 +3,21 @@
 import numpy as np
 import orjson
 import pandas as pd
-import plotly.express as px
-import plotly.io as pio
 from torch import Tensor
 
-pio.kaleido.scope.mathjax = None  # https://github.com/plotly/plotly.py/issues/3469
+
+def import_plotly():
+    try:
+        import plotly.express as px
+        import plotly.io as pio
+    except ImportError:
+        raise ImportError(
+            "Plotly is not installed.\n"
+            "Please install it using `pip install plotly`, "
+            "or install the `[visualize]` extra."
```

Comment:
optional nit: maybe `... install this repo with the [visualize] extra` for clarity